src/node/node_state.h

```diff
@@ -67,6 +67,9 @@ namespace ccf
   using RaftType =
     raft::Raft<consensus::LedgerEnclave, NodeToNode, Snapshotter>;
 
+  static const std::string IP_ADDRESS_PREFIX("iPAddress:");
```

Comment:
Duplicated in `cli_helper.h`.

Reply:
Yes, forgot to remove it when it moved, thanks for spotting it.